deploy/caf-secure-deploy.json

```
@@ -2,44 +2,136 @@
     "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "resourceGroupName": {
-            "type": "String",
+        "deploymentPrefix": {
+            "type": "string",
+            "maxLength": 10,
             "metadata": {
-                "displayName": "Resource group name",
-                "description": "Provide name for resource group"
+                "displayName": "Deployment prefix",
+                "description": "Provide a prefix (max 10 characters, unique at tenant-scope) for subscription resources created as part of this deploy."
             }
         },
-        "workspaceName": {
+        "emailSecurityContact": {
             "type": "String",
             "metadata": {
-                "displayName": "Log Workspace name",
-                "description": "Provide name for log analytics workspace"
+                "displayName": "Azure Security Center Email Contact",
+                "description": "Email address to get email notifications from Azure Security Center"
             }
         },
+        "enableCISInitiative": {
+            "type": "string",
+            "metadata": {
+                "displayName": "Enable CIS Microsoft Azure Foundations Benchmark initiative",
+                "description": "If 'Yes', ARM will assign the CIS Microsoft Azure Foundations Benchmark initiative to the subscription. If 'No', it will be skipped"
+            },
+            "defaultValue": "Yes",
+            "allowedValues": [
+                "Yes",
+                "No"
+            ]
+        },
         "retentionInDays": {
-            "type": "String",
+            "type": "int",
             "metadata": {
                 "displayName": "Data retention (days)",
                 "description": "Select data retention (days) for Log Analytics."
             },
             "defaultValue": "30"
```

does this go to 
```suggestion
            "defaultValue": 30
```
now that it's an `int`?